packages/ng/dialog/dialog/dialog.component.ts

```diff
@@ -0,0 +1,35 @@
+import { ChangeDetectionStrategy, Component, ElementRef, inject, OnInit, ViewEncapsulation } from '@angular/core';
+import { LuDialogRef } from '../model';
+
+@Component({
+	selector: 'lu-dialog',
+	standalone: true,
+	template: '<ng-content></ng-content>',
+	styleUrl: './dialog.component.scss',
+	changeDetection: ChangeDetectionStrategy.OnPush,
+	encapsulation: ViewEncapsulation.None,
+	host: {
+		class: 'dialog-inside',
+	},
+})
+export class DialogComponent implements OnInit {
+	public readonly dialogRef = inject<LuDialogRef>(LuDialogRef);
+
+	#htmlElement = inject<ElementRef<HTMLElement>>(ElementRef).nativeElement;
+
+	close(): void {
+		this.dialogRef.close();
+	}
+
+	dismiss(): void {
+		this.dialogRef.dismiss();
+	}
+
+	ngOnInit(): void {
+		if (!this.dialogRef.config.cdkConfigOverride?.autoFocus) {
```

Comment:
By default, the focus should be on the close button. You want to be able to override it, but it must remain an exception that you activate deliberately.